bitnami/spark/README.md

```diff
@@ -0,0 +1,256 @@
+| `security.ssl.protocol`                     | Set the SSL protocol                                                                        | `TLSv1.2`                                               |
+| `security.certificatesSecretName`           | Set the name of the secret that contains the certificates                                   | No default                                              |
+| `service.type`                              | Kubernetes Service type                                                                     | `ClusterIP`                                             |
+| `service.port`                              | spark client port                                                                           | `2379`                                                  |
+| `service.nodePort`                          | Port to bind to for NodePort service type (client port)                                     | `nil`                                                   |
+| `service.annotations`                       | Annotations for spark service                                                               | {}                                                      |
+| `service.loadBalancerIP`                    | loadBalancerIP if spark service type is `LoadBalancer`                                      | `nil`                                                   |
+| `ingress.enabled`                           | Enable the use of the ingress controller to access the web UI                               | `true`                                                  |
+| `ingress.hosts`                             | Add hosts to the ingress controller with name and path                                      | `name: spark.local`, `path: /`                          |
+
+
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install --name my-release \
+  --set master.webPort=8081 bitnami/spark
+```
+
+The above command sets the spark master web port to `8081`.
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install --name my-release -f values.yaml bitnami/spark
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+### Production configuration and horizontal scaling
+
+This chart includes a `values-production.yaml` file where you can find some parameters oriented to production configuration in comparison to the regular `values.yaml`.
+
+```console
+$ helm install --name my-release -f ./values-production.yaml bitnami/spark
+```
+
+- Enable RPC authentication:
+```diff
+- security.rpc.authenticationEnabled: no
++ security.rpc.authenticationEnabled: yes
+```
+
+- Enable RPC encryption:
```

Comment:
You can merge this one in the previous one

Reply:
Why? One is authentication and the other is encryption.

Reply:
- Enable RPC authentication and encryption:
```diff
- security.rpc.authenticationEnabled: no
- security.rpc.encryptionEnabled: no
+ security.rpc.authenticationEnabled: yes
+ security.rpc.encryptionEnabled: yes
```